### DISCUSSION TITLE
R68S Docker 国际版固件下载

### DISCUSSION THREAD

**Discussion by coolsnowwolf** [coolsnowwolf/lede#9509] [R68S / 电犀牛专区]
R68S Docker 国际版固件下载

链接: https://pan.baidu.com/s/1JBL_BpUzAT3D9x8PgkznNg 提取码: bhfy

**Comment by xielucky** [comment]
不想刷固件,软件可以打包么

**Reply by shaufo** [reply]
很好，很强大。折腾好久都没能访问的netdata终于集成了。

**Comment by xielucky** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by shaufo** [comment]
希望L大下版固件多集成一些插件。虽然不一定能用上，例如：adguard home；多线多拨等等，听说过，没弄过。自己安装也安装不上，能集成最好。想起个笑话，某位伟人说过“我可以不用，但是不能没有”

**Reply by nandehututu** [reply]
想想是不是鲁树人说的 （笑

**Comment by wenzi202** [comment]
大雕，请问单线多帐号多拨后，怎么设置分流实现cdn业务固定IP?    多拨后看B站快进要卡顿一秒。

**Comment by kiritoxjf** [comment]
带Docker的固件我都试过了，WAN口拨通了，LAN口就是不能上网，不知道为啥，有没有老哥遇到这个情况？怎么解决的啊？

**Reply by kiritoxjf** [reply]
目前只有不带Docker的固件可以稳定使用，但是不带Docker的固件软件源里又没有Dockerd守护程序，没法自己安装Docker，就很麻

**Comment by wenzi202** [comment]
检查电脑自动获取IP是不是169开头的，我遇到了，手动配置192.168.100.**



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2022年06月12日 13:19 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.******@***.***> |
| 主题 | Re: [coolsnowwolf/lede] R68S Docker 国际版固件下载 (Discussion #9509) |

带Docker的固件我都试过了，WAN口拨通了，LAN口就是不能上网，不知道为啥，有没有老哥遇到这个情况？怎么解决的啊？

**Comment by mickeynuo** [comment]
请教L大，r68s如何设置链路聚合。就是把两条宽带链路到一起，实现网速叠加。

**Reply by code33** [reply]
这玩意的故障是莫名其妙的多,不建议玩它

**Comment by xielucky** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by xielucky** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
